frontend/src/components/Login.vue

```diff
@@ -54,14 +54,14 @@
 
 <script>
 import { minLength, required } from 'vuelidate/lib/validators';
+import { mapGetters } from 'vuex';
 
 export default {
   data() {
     return {
       username: '',
       password: '',
       showPass: false,
```

Comment:
Can we rename it to showPassword? I needed to look for it to understand what it does.

Reply:
Done